scripts/apps/templates/directives/TemplatesDirective.js

```diff
@@ -269,11 +269,29 @@ export function TemplatesDirective(gettext, notify, api, templates, modal, desks
             });
 
             function setupContentType(type) {
-                $scope.schema = content.schema(type, $scope.item.type);
-                $scope.editor = content.editor(type, $scope.item.type);
+                if (type) {
+                    $scope.schema = content.schema(type, $scope.item.type);
+                    $scope.editor = content.editor(type, $scope.item.type);
+                }
             }
 
             $scope.remove = function(template) {
+                let deskTemplates = _.filter($scope.desks._items, (desk) =>
+                        desk.default_content_template === template._id),
+                    deskNames = ' ';
+
+                if (!_.isEmpty(deskTemplates)) {
+                    _.forEach(deskTemplates, (desk, index) => {
```

Comment:
I would prefer functional style here:
```js
const deskNames = deskTemplates.map((desk) => desk.name).join(', ');
```
and then when concatenating the strings:
```js
` ${deskNames}.` +
```